_search-plugins/knn/approximate-knn.md

```diff
@@ -81,7 +81,7 @@ PUT my-knn-index-1
 
 In the example above, both `knn_vector` fields are configured from method definitions. Additionally, `knn_vector` fields can also be configured from models. You can learn more about this in the [knn_vector data type]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index#knn_vector-data-type) section.
 
-The `knn_vector` data type supports a vector of floats that can have a dimension of up to 10000 for the nmslib and faiss engines, as set by the dimension mapping parameter. The maximum dimension for the Lucene library is 1024.
+The `knn_vector` data type supports a vector of floats that can have a dimension of up to 16000 for the nmslib and faiss engines, as set by the dimension mapping parameter. The maximum dimension for the Lucene library is 1024.
```

Comment:
Should it be, " .. that can have up to 16,000 dimensions for the nmslib..."? And "The maximum number of dimensions for the Lucene bibrary is 1024."

Reply:
@kolchfa-aws This could impact the definition of what dimension means in this context. Right now, it reflects a single parameter. If we pluralize it, it may change the meaning.

Reply:
@macrakis Can you confirm that keeping the current wording is accurate?